### DISCUSSION TITLE
什么时候更换成nftables啊

### DISCUSSION THREAD

**Discussion by Vbaethon** [istoreos/istoreos#1316] [Ideas]
nftables好一点吧， 现在用的一些常用的网站都解析有问题。很不方便

**Comment by jjm2473** [comment]
网站解析跟防火墙有什么关系？
据我所知，有些游戏加速软件，还有“应用过滤”都不支持nftables

**Reply by jjm2473** [reply]
相关讨论在升级22.03的时候就已经有了 https://github.com/istoreos/istoreos/issues/716
